news/2023/2023-11-03-the-followpoint-ephemeral-the-man-behind-the-medals.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "The Followpoint: Ephemeral, The Man Behind The Medals"
+date: 2023-11-03 00:00:00 +0000
+---
+
+Come join us to delve into the science and magnets behind medals with Ephemeral, osu!'s strongest seagull!
+
+![](/wiki/shared/news/2023-11-03-the-followpoint-ephemeral-the-man-behind-the-medals/banner.jpg)
+
+Welcome to **The Followpoint**, a brand new series of interview-style newsposts (and podcasts), with a different guest each release! This time around we were joined by [Ephemeral](https://osu.ppy.sh/users/102335), the swole seagull who is in charge of all the magic behind medals!
+
+Ephemeral, joining the game in August 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has the role of Community Manager and leads medal development. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished community member.
```

Comment:
shouldn't be copying https://osu.ppy.sh/home/news/2013-07-31-interview-with-ephemeral

```suggestion
Ephemeral is a bit of an ever-present figure in osu!. Players as ancient as him may remember him for making [one of the (top 200) most played beatmaps of all time](https://osu.ppy.sh/beatmapsets/10435), but most people know him for being a "community manager", having a hand in many things going on in the back-end.

Among his tasks is leading the development of medals. This makes him the prime subject of avid medal hunters, who have decided to organise an interview where anyone could attend to ask any questions their soul desired answers for, regarding medals or the man himself.
```